app/backend/wells/serializers.py

```diff
@@ -67,6 +68,22 @@ class Meta:
         }
 
 
+class DecommissionDescriptionSerializer(serializers.ModelSerializer):
+    """Serializes Decommission Descriptions"""
+
+    class Meta:
+        model = DecommissionDescription
+        fields = (
+            'decommission_description_guid',
```

Comment:
Technically we don't need this right now; but if we need this in the future depends on if we're going to "stack" de-commission alterations.